package.json

```diff
@@ -30,7 +30,7 @@
     "raf": "^3.3.2",
     "ric-shim": "^1.0.0",
     "smoothscroll-polyfill": "^0.3.5",
-    "webrtc-adapter": "^2.1.0",
+    "webrtc-adapter": "^2.1.0 || ^3 || ^4 || ^5",
```

Comment:
Why don't we pick one?

Reply:
So we don't introduce a breaking change! @ljharb

Reply:
@goatslacker I talked to @ljharb about this before making the PR, and I think the hope is that we don't introduce breaking changes to `browser-shims` due to deprecations in `webrtc-adapter` so we don't have to update it to `semver-major`? Is that right @ljharb? Let me know if I misunderstood.